src/Components/Publishing/Layouts/__tests__/StandardLayoutAdsEnabled.test.tsx

```diff
@@ -45,4 +47,116 @@ describe("Standard Article with new ads hidden behind feature flags", () => {
     expect(article.find(DisplayPanel).length).toBe(0)
     expect(article.find(DisplayCanvas).length).toBe(0)
   })
+
+  it("It does not render new panel and canvas displays in Standard Layout Articles when feature flagged ads are disabled", () => {
+    standardArticleProps.areHostedAdsEnabled = false
+    const article = getWrapper(standardArticleProps)
+
+    expect(article.find(NewDisplayPanel).length).toBe(0)
+    expect(article.find(NewDisplayCanvas).length).toBe(0)
+    expect(article.find(DisplayPanel).length).toBe(1)
+    expect(article.find(DisplayCanvas).length).toBe(1)
+  })
+})
+
+describe("data", () => {
+  it("renders the canvas component with the correct data and properties on standard articles", () => {
+    const canvas = mount(
+      <NewDisplayCanvas
+        adDimension={AdDimension.Desktop_TopLeaderboard}
+        adUnit={AdUnit.Desktop_TopLeaderboard}
+        displayNewAds
+        targetingData={targetingData(StandardArticle.id, "article")}
+      />
+    )
+
+    expect(canvas.props().adDimension).toEqual("970x250")
+    expect(canvas.props().adUnit).toEqual("Desktop_TopLeaderboard")
+    expect(canvas.props().displayNewAds).toBe(true)
+    expect(canvas.props().targetingData).toEqual({
+      is_testing: true,
+      page_type: "article",
+      post_id: "594a7e2254c37f00177c0ea9",
+    })
+    expect(canvas).toHaveLength(1)
+    canvas.find({ className: "htl-ad" })
```

Comment:
Where is this `htl-ad` class coming from? Also you're not doing anything with the result of `.find()`

Reply:
Good find, I think is left over from when I first started working in this and wasn't sure what to assert. I just removed all instances of them.  ` htl-ad` was the class name of the div before you implemented the React GPT, so either way it should be removed.